diff -aurN gnome-cups-manager-0.18.orig/gnome-cups-add/snmpkit/snmpkit_interface.C gnome-cups-manager-0.18/gnome-cups-add/snmpkit/snmpkit_interface.C
--- gnome-cups-manager-0.18.orig/gnome-cups-add/snmpkit/snmpkit_interface.C	2003-05-17 21:01:47.000000000 +0200
+++ gnome-cups-manager-0.18/gnome-cups-add/snmpkit/snmpkit_interface.C	2004-06-07 00:43:54.530736632 +0200
@@ -508,7 +508,7 @@
 void **sk_table_get(SNMPTABLE *st){
   std::list<void*> vals;
   from_c(st)->get(vals);
-  void **retval=new (void*)[vals.size()+1];
+  void **retval=new void*[vals.size()+1];
   retval[vals.size()]=NULL;
   int i=0;
   for(std::list<void*>::iterator cur=vals.begin();cur!=vals.end();cur++,i++)
