--- gnome-cups-manager-0.24/configure.in.orig	2004-08-19 04:36:11.000000000 +0200
+++ gnome-cups-manager-0.24/configure.in	2004-08-24 23:04:25.593601496 +0200
@@ -83,7 +83,7 @@
 AC_PROG_INTLTOOL([0.20])
 
 dnl Add the languages which your application supports here.
-ALL_LINGUAS="bg ca cs de el en_CA en_GB es fi fr he hr hu it ja ko nb nl no pa pt pt_BR sr sr@Latn sv tr uk"
+ALL_LINGUAS="bg ca cs de el en_CA en_GB es fi fr he hr hu it ja ko nb nl pa pt pt_BR sr sr@Latn sv tr uk"
 AM_GLIB_GNU_GETTEXT
 
 AC_OUTPUT([
