diff -urN aa/gnome-cups-manager-0.25/gnome-cups-manager/gnome-printer-list.c gnome-cups-manager-0.25/gnome-cups-manager/gnome-printer-list.c
--- aa/gnome-cups-manager-0.25/gnome-cups-manager/gnome-printer-list.c	2004-09-13 22:00:30.000000000 +0200
+++ gnome-cups-manager-0.25/gnome-cups-manager/gnome-printer-list.c	2004-09-13 22:54:57.084780440 +0200
@@ -345,7 +345,7 @@
 	gtk_widget_show (item);
 	gtk_menu_shell_append (GTK_MENU_SHELL (menu), item);	
 
-	item = gtk_image_menu_item_new_from_stock (GTK_STOCK_MEDIA_PAUSE, NULL);
+	item = gtk_image_menu_item_new_with_label (_("Pause"));
 	gtk_widget_show (item);
 	gtk_widget_set_sensitive (item, pause);
 	g_signal_connect (item, "activate", 
